pkg/annotations.go

```diff
@@ -254,3 +258,12 @@ func (cronitorParser CronitorConfigParser) GetGraceSeconds() int {
 	}
 	return -1
 }
+
+func (cronitorParser CronitorConfigParser) EnabledAutoComplete() (bool, error) {
```

Comment:
the default behavior needs to be that this is enabled. Unless I'm misunderstanding this defaults it to not sending the Complete ping. Please update this to default to true and add a test for it.